### ISSUE TITLE
Project manager should declare export formats

### ISSUE THREAD

**Issue by simoncozens** [googlefonts/fontra#1878]
Currently if the project manager supplies `export-as` in its info, then Fontra assumes that it supports *all* export formats:

https://github.com/googlefonts/fontra/blob/23f45d4008e6a589a8036adcea46fb0a5e68817f/src/fontra/views/editor/editor.js#L696-L699

An alternative is for Fontra to read the `projectManagerFeatures["export-as"]` as a list of supported export formats. Happy to PR this if it's a good idea.

**Comment by justvanrossum** [comment]
Yeah, good point, and good solution. Please keep the relationship with Fontra Pak in mind: https://github.com/googlefonts/fontra-pak/blob/main/FontraPakMain.py

**Comment by simoncozens** [comment]
So I'll probably change:

```
        for key, methodName in [("export-as", "exportAs")]:
            projectManagerFeatures[key] = hasattr(self.projectManager, methodName)
```

to

```
        if hasattr(self.projectManager, "exportAs"):
            projectManagerFeatures["export-as"] = self.projectManager.supportedExportFormats
```

FileSystemProjectManager doesn't support any, AFIACS, and I'll send a separate PR to subclass it in FontraPak.

While I'm about it I'll try to fix up the fact that nothing checks whether `features["find-glyphs-that-use-glyph"]` before using it...
